src/lit-html.ts

```diff
@@ -528,6 +528,11 @@ export class NodePart implements SinglePart {
 
   setValue(value: any): void {
     value = getValue(this, value);
+    if (value === '$NO_OP') {
```

Comment:
Rather than a magic string, I'd like to export a special value, like we do for `directiveValue`. In fact, `directiveValue` might be able to serve as this value already. Maybe it just needs a new name.

Reply:
What about a symbol?

Reply:
It doesn't really matter - any unique object will work - except that IE11 doesn't have Symbols.

Reply:
yeah looks like directiveValue is handling it as well :)
what about renaming directiveValue to alreadyHandled?

Reply:
I think something like `noChange` might be a better name.

Reply:
fine for me... lemme know if i should create a new pr

Reply:
I was running into the same issue today, and wasn't able to find the `directiveValue` solution. Changing the name to `noChange` sounds good to me